src/components/shared/Entity/Status.tsx

```diff
@@ -18,7 +20,11 @@ function Status() {
     }
 
     if (messageKey) {
-        return <FormattedMessage id={messageKey} />;
+        return (
+            <Typography>
```

Comment:
**Merge Conflict Source:** With your changes, the margin I added to this component can be safe overwritten.